bridge/client/app/project-board/project-board.component.html

```diff
@@ -50,11 +50,31 @@
                                    fxFlex="0 1 calc(33.3% - 15px)" fxFlex.lt-md="0 1 calc(50% - 15px)" fxFlex.lt-sm="100%"
                                    (click)="selectStage(stage)" [selected]="selectedStage == stage">
                 <h2 class="m-0 mt-1 mb-1" [textContent]="stage.stageName"></h2>
-                <hr/>
-                <ng-container *ngIf="stage.services">
+                <div class="stage-state" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="15px">
+                  <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="5px">
+                    <dt-icon class="stage-state-icon" name="criticalevent"></dt-icon>
+                    <span>0</span>
+                  </div>
+                  <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="5px">
+                    <dt-icon class="stage-state-icon" name="traffic-light"></dt-icon>
+                    <span>0</span>
+                  </div>
+                  <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="5px">
+                    <ng-container *ngIf="openApprovals$ | async as openApprovals">
+                      <dt-icon class="stage-state-icon" name="deploy" [class.out-of-sync-service]="countOpenApprovals(openApprovals, stage) > 0"></dt-icon>
+                      <span [textContent]="countOpenApprovals(openApprovals, stage)"></span>
+                    </ng-container>
+                  </div>
+                </div>
+                <ng-container *ngIf="stage.services && stage.services.length > 0; else noService">
                   <dt-tag-list aria-label="services">
-                    <dt-tag *ngFor="let service of getDeployedServices(project, stage)">
-                      <span *ngIf="getLatestDeployment(project, service, stage) as deployment" (click)="selectDeployment(deployment, project)" [textContent]="deployment.getShortImageName()"></span>
+                    <dt-tag *ngFor="let service of stage.services">
+                      <ng-container *ngIf="openApprovals$ | async as openApprovals">
+                        <span *ngIf="project.getLatestDeployment(service, stage) as deployment; else noDeploymentOfServiceTag" [ngClass]="{'out-of-sync-service': countOpenApprovals(openApprovals, stage, service) > 0}" (click)="selectDeployment(deployment, project)" [textContent]="deployment.getShortImageName()"></span>
```

Comment:
this line is to large maybe this way is more visible:
<span *ngIf="project.getLatestDeployment(service, stage) as deployment; else noDeploymentOfServiceTag" 

[ngClass]="{'out-of-sync-service': countOpenApprovals(openApprovals, stage, service) > 0}" 

(click)="selectDeployment(deployment, project)" 

[textContent]="deployment.getShortImageName()"><//span>

Reply:
I split the code line into multiple lines. There is also a follow-up issue, to split the project-board into smaller components #1989